src/OfficeToolPlus/Dictionaries/Languages/ru-ru.xaml

```diff
@@ -6,52 +6,52 @@
     <!-- Country settings, it is used to get information from server. -->
     <sys:String x:Key="Culture">ru-ru</sys:String>
     <!-- Products name settings, "en-us" is default value if not set. -->
-    <sys:String x:Key="ProductDict">en-us</sys:String>
+    <sys:String x:Key="ProductDict">ru-ru</sys:String>
```

Comment:
To change this, please translate product name to Russian first. Here is the file location: `https://github.com/YerongAI/Office-Tool/tree/main/src/OfficeToolPlus/Dictionaries/ProductsName`

Reply:
Returned. I misunderstood the parameter description.